tests/locals.tf

```diff
@@ -0,0 +1,10 @@
+locals {
+  # Name, name tag, log name related locals
```

Comment:
is this comment still relevant/accurate?

Reply:
nope, removed